dlp/snippets/inspect_content_test.py

```diff
@@ -19,12 +19,12 @@
 import backoff
 import google.api_core.exceptions
 from google.api_core.exceptions import ServiceUnavailable
-import google.cloud.bigquery
-import google.cloud.datastore
+# import google.cloud.bigquery
+# import google.cloud.datastore
 import google.cloud.dlp_v2
-import google.cloud.exceptions
-import google.cloud.pubsub
-import google.cloud.storage
+# import google.cloud.exceptions
```

Comment:
Please remove comments. Also, if these dependencies are included in `requirements.txt` or `requirements-test.txt`, they should be removed there, too.

Reply:
they are part of the code, I had commented those locally (as I was facing some issue ) to test things. Mistakenly I pushed the comments. Reverted now. thank you for your comment.